chef_master/source/berkshelf.rst

```diff
@@ -3,7 +3,181 @@ About Berkshelf
 =====================================================
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/berkshelf.rst>`__
 
-Berkshelf is a dependency manager for certain cookbook workflows that is included in the Chef development kit. Berkshelf stores every version of a cookbook that has ever been installed. By default, the local disk repository for Berkshelf is located at ``~/.berkshelf``, but this may be modified by setting the ``BERKSHELF_PATH`` environment variable.
+.. tag berkshelf_summary
+
+Berkshelf is a dependency manager for certain cookbook workflows that is included in the Chef development kit.
+
+.. note:: For new users, we strongly recommend using :doc:`Policyfiles </policyfile>` rather than Berkshelf.
+
+.. end_tag
+
+The Berksfile
+==============
+
+.. tag berksfile_summary
+
+A Berksfile describes the set of sources and dependencies needed to use a cookbook. It is used in conjunction with the ``berks`` command.
+
+.. end_tag
+
+Syntax
+-------
+A Berksfile is a ruby file, in which sources, dependencies, and options may be specified. Berksfiles are modelled closely on Bundler's Gemfile. The syntax is as follows:
+
+.. code-block:: ruby
+
+  source "https://supermarket.chef.io"
+  metadata
+  cookbook "NAME" [, "VERSION_CONSTRAINT"] [, SOURCE_OPTIONS]
+
+
+Metadata Keyword
```

Comment:
Shouldn't the metadata and source section be swapped here so we follow the order of the Berksfile?